operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -305,3 +305,15 @@ The following example is a block of Go code.
 ## Using Markdown
 
 The majority of Mattermost technical documentation is written in `.rst`. However, there are some instances where Markdown is used, for example the Mattermost Handbook. You can read more about using Markdown in the [Formatting Text](https://docs.mattermost.com/help/messaging/formatting-text.html) section of the User's Guide.
+
+## Editing the Mattermost User Guide
+
+These are some additional tips for writing [Mattermost documentation for end users](https://docs.mattermost.com/guides/user.html).
+
+1.	Start fresh - begin by writing a draft in a Google document from scratch.
```

Comment:
Is the added indentation on purpose?

Reply:
Nope, definitely not. Thanks for catching it - it only showed up when I was in "suggestion" mode for some reason.